README.md

```diff
@@ -27,76 +27,81 @@ Set global options in a `keyManagerConfig` object in your application's `config/
 | ---------- | ------------- | -------- | -------- | --------- |
-| `keys`     | array of strings | Yes      | `<none>` | An array of keys that comprise the shortcut e.g., `['meta', 'enter']`. If `keys` contains multiple modifier keys, they still comprise one shortcut e.g., `['meta', 'shift', 'enter']` is a single shortcut that executes when all three of those keys are pressed. |
-| `name`     | string | Yes      | `<none>` | A unique string by which you can identify – and `deregister` – the shortcut. |
-| `selector`     | jQuery selector | No      | `$(document)` | A jQuery selector by which to scope the shortcut. |
-| `downCallback`     | function | No      | `<none>` | A function to be called when the shortcut keys are matched and the keydown event is fired. |
-| `upCallback`     | function | No      | `<none>` | A function to be called when the shortcut keys are matched and the keyup event is fired. |
-| `priority`     | integer | No      | `0` | An integer used to prioritize shortcuts should the shortcuts with the same `keys` be bound at the same time. For example, you bind the `escape` key on a route and the route's template renders a component that also binds the `escape` key. Highest priority takes precedence. |
-| `disableOnInput`     | boolean | No      | `false` | A boolean to determine whether the callback should be fired when an `input`, `textarea`, or `select` element is active.
+| `callback` | `Function` | Yes | `null` | A function to be called when the macro keys are matched, the `keyEvent` event is fired, and the scope defined by `element` is correct. When called, the `callback` is called with the keyboard `event` that triggered the macro. |
+| `element` | `Element` | No | `document.body` | A DOM element by which to scope the macro. |
+| `executionKey` | `String` | Yes | `''` | A string that's the value of the key that triggers the macro's callback e.g., to make letter A the execution key, set `executionKey` to `a`. If unsure of a key's value, [use this tool](https://codepen.io/patrickberkeley/full/PEexPY) to it test out. |
+| `isDisabledOnInput` | `Boolean` | No | `false` | A boolean to determine if a macro's the callback should be fired when a `contentEditable`, `input`, `textarea`, or `select` element is focused. |
+| `modifierKeys` | `Array` | No | `[]` | An array of modifier key names. Options are `Alt`, `Control`, `Meta`, `Shift`. |
+| `priority` | `Number` | No | `0` | An integer used to prioritize macros if there's more than one macro with the same `keys` listening at the same time. For example, you add a macro with the `escape` key on a route and the route's template renders a component that also binds a macro with the `escape` key. Highest priority takes precedence. |
+| `keyEvent` | `String` | Yes | `null` | Dictates which key event is used for the macro. Options are: `keydown`, `keyup`.
 
-#### `deregister({name})`
+#### `removeMacro({ element, keyEvent })`
```

Comment:
this feels misleading - as you actually are passing in the macro object that is returned in `addMacro`.

Reply:
Doesn't have to be the macro though. Just any object with the correct element and key event name. Still thing I should change it?

Reply:
Doesn't it need executionKeys and modifierKeys in order to find the correct macro to remove?

Reply:
Actually yeah that is a mistake with the readme…